custom/icds/messaging/indicators.py

```diff
@@ -118,9 +104,10 @@ def get_messages(self, language_code=None):
             'awc': self.user.sql_location.name,
         }
 
-        message = self.render_template(context, language_code=language_code)
+        if more_than_one_week or more_than_one_month:
```

Comment:
it would be slightly more performant to do this check before we set the `context` and return `[]` if both are False, so that we don't have to evaluate `self.user.sql_location`